src/Microsoft.Azure.WebJobs.Storage/Blob/StorageBlobClient.cs

```diff
@@ -58,6 +58,16 @@ public Task<ServiceProperties> GetServicePropertiesAsync(CancellationToken cance
             BlobListingDetails blobListingDetails, int? maxResults, BlobContinuationToken currentToken,
             BlobRequestOptions options, OperationContext operationContext, CancellationToken cancellationToken)
         {
+            // Add WebJobs to user agent to make it easier to collect usage information from Storage telemetry
+            if (operationContext == null)
```

Comment:
Is there a way we can specify this globally on the clients themselves (CloudBlobClient/CloudQueueClient/etc.). E.g some client level event, or perhaps via https://msdn.microsoft.com/en-us/library/microsoft.windowsazure.storage.blob.cloudblobclient.setserviceproperties(v=azure.10).aspx?